sentry_dart_plugin/lib/src/utils/log.dart

```diff
@@ -0,0 +1,105 @@
+import 'dart:io';
+import 'package:ansicolor/ansicolor.dart';
+
+int numberOfAllTasks = 11;
+
+class Log {
+  static AnsiPen red = AnsiPen()..red(bold: true);
+  static AnsiPen yellow = AnsiPen()..yellow(bold: true);
+  static AnsiPen green = AnsiPen()..green(bold: true);
+  static AnsiPen blue = AnsiPen()..blue(bold: true);
+  static AnsiPen gray05 = AnsiPen()..gray(level: 0.5);
+  static AnsiPen gray09 = AnsiPen()..gray(level: 0.9);
```

Comment:
`l`: Maybe we can use Sentry colors here.

Reply:
that would be nice, indeed, i just copied this log class from another repo that Ive inspired from

Reply:
Don't we have to mention that we copied the class, because of the license?

Reply:
its MIT so don't think so but I've added a comment about it since I've copied only a part of it